pickATrip_django_apps/templates/pickATrip_django_apps/base.html

```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE html>
+{% load socialaccount %}
```

Comment:
Did you check base, once you are logging with local user?
I would recommend to rethink and suggest a test

Reply:
are you asking whether this change harming the logging process of users not through the google-sign-in functionality?

Reply:
right now I'm having trouble solving conflicts on the Pipfile.lock, I usually don't change it manually and trying to use the 'pipenv lock --clear', 'pipenv sync --dev' and 'pipenv lock' to generate the Pipfile.lock from the existing Pipfile, but its not working and I tried everything that used to wor, can't find solutions on google that is working for me. (It is throwing error about black and then after erasing it now it throws the same error on the flake8), should I generate it from scratch maybe?